### DISCUSSION TITLE
채팅 통신 오류 관련 문의

### DISCUSSION THREAD

**Discussion by LeeYongSu1** [naverz/zepeto-studio-kor#535] [5. Q&A]
World Ver : 1.1.5

App Ver: 3.10.4

채팅이 바로 바로 올라가지 않고, 채팅이 일정 시간 후에 보이거나, 글 순서가 바뀌는 등의 문제가 있습니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 현재 출시된 맵에서는 재현이 안되고 있는데
어떤 맵인지 상세하게 알려주시면 도움이 될 것 같습니다.

**Reply by LeeYongSu1** [reply]
월드명: 트래블버스
입니다

**Reply by Zepeto-TechEvangelist** [reply]
넵 확인해보겠습니다.

**Reply by Zepeto-TechEvangelist** [reply]
뉴월드 채팅에 딜레이가 있는 부분 확인되었고 추후 개선 예정입니다. 
감사합니다.

**Reply by LeeYongSu1** [reply]
네 감사합니다!
